examples/image_classification/README.md

```diff
@@ -0,0 +1,393 @@
+var viewSpec = new TbTaskViewSpec(Map.of());
+viewSpec.setConfig(config);
+var lock = Map.of("core", "1.0.0",
+        "view.image", "1.0.0",
+        "condition.required", "1.0.0",
+        "field.button-radio-group", "1.0.0",
+        "view.list", "1.0.0",
+        "plugin.toloka", "1.0.0",
+        "action.set", "1.0.0",
+        "plugin.hotkeys", "1.0.0");
+        viewSpec.setLock(lock);
+TaskSpec taskSpecification = new TaskSpec(inputSpec, outputSpec, viewSpec);
+```
+
+Write short and simple instructions.
+
+```java
+String instructions = "<p>Decide what category the image belongs to.</p>\n" +
+        "<p>Select \"<b>Cat</b>\" if the picture contains one or more cats.</p>\n" +
+        "<p>Select \"<b>Dog</b>\" if the picture contains one or more dogs.</p>\n" +
+        "<p>Select \"<b>Other</b>\" if:</p>\n" +
+        "<ul><li>the picture contains both cats and dogs</li>\n" +
+        "<li>the picture is a picture of animals other than cats and dogs</li>\n" +
+        "<li>it is not clear whether the picture is of a cat or a dog</li>\n" +
+        "</ul>";
+```
+
+Create a project.
+
+```java
+Project project = new Project(name, description, instructions, taskSpecification, AssignmentsIssuingType.AUTOMATED);
+ModificationResult<Project> projectCreationResult = clientFactory.getProjectClient().createProject(project);
+String projectId = projectCreationResult.getResult().getId();
+```
+
+![created project](img/created_project.png)
+
+Link to open in web interface: `https://toloka.yandex.com/requester/project/<project_id>`
+
+
+## Pool creation
+
+Specify the [pool parameters](https://yandex.com/support/toloka-requester/concepts/pool_poolparams.html).
+
+```java
+Pool pool = new Pool(projectId, 
+        "private_name",                                      // Give the pool any convenient name. You are the only one who will see it.
+        false,                                               // Specify, if pool may contain adult content
+        Date.from(Instant.now().plus(365, ChronoUnit.DAYS)), // Set expiration date
+        BigDecimal.valueOf(0.01),                            // Set the price per task page.
+        600,                                                 // Time allowed for completing a task page
+        true,                                                // Allow auto-acceptence for solutions or not
+        new PoolDefaults(3)                                  // Overlap. This is the number of users who will complete the same task.
+);
+```
+
+Select English-speaking performers.
+
+```java
+pool.setFilter(new Connective.And(List.of(new Expression.Languages(ArrayInclusionOperator.IN, LangIso639.EN))));
+```
+
+Set up [Quality control](https://yandex.com/support/toloka-requester/concepts/control.html). Add basic controls. 
+And Golden Set aka Control tasks. Ban performers who give incorrect responses to control tasks.
+
+```java
+RuleConfig incomeSumRuleConfig = new RuleConfig(
```

Comment:
Lets have description for each rule config, so users understand what it does

Reply:
Done